packages/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Ignore most of NuGet package structure, except DLLs which are required by Unity.
+/*/*/*
+
+!/*/*/analyzers
+!/*/*/analyzers.meta
+
+!/*/*/lib
+!/*/*/lib.meta
```

Comment:
is the number of subdirectories important? Do we maybe want to ignore something like `/**` and then include `.dll`s, or something?

Reply:
Unfortunately, that doesn't work in Git. It's a common issue and the workaround is always the same - you need to explicitly write out the path components instead of using `**` :(

If you find a better way, let me know though.

Reply:
Wait, what do you mean it doesn't work? When I replace the first line with `/**` it still seems to ignore the same files? (and when I remove it, a bunch of new files show up)

Reply:
sure, but `analyzers` and `lib` are in a specific place. it makes sense for those to be un-ignored at a specific depth. It makes less sense to me to ignore everything only up to depth 3 though.

not a big deal :shrug:

Reply:
> It makes less sense to me to ignore everything only up to depth 3 though.

I agree, I'm just saying that - AFAIK - it's not possible to ignore `/**` but then unignore specific depths.

E.g. see similar situation and solution here: https://stackoverflow.com/a/28489689/439965

Reply:
Oh gotcha, I misunderstood. Makes sense!